bittide/src/Bittide/ClockControl/Si539xSpi.hs

```diff
@@ -225,6 +227,8 @@ data DriverState dom = DriverState
   -- ^ Current communication transaction.
   , commandAcknowledged :: Acknowledge
   -- ^ Whether or not the current transaction has already been acknowledged.
+  , storedBytes :: Maybe Byte
+  -- ^ Data we have received from the SPI interface.
```

Comment:
The name `storedBytes` (plural) does not match the type `Byte` (singular). But the commit message calls it `storedWord`. So what type should it be?

Reply:
Just discussed this in the office: `storedByte`.